examples/.experimental/next-app-dir/src/app/server-action/FormWithUseActionExample.tsx

```diff
@@ -9,7 +9,7 @@ export function FormWithUseActionExample() {
     <>
       <p>Check the console for the logger output.</p>
       <form
-        action={testAction}
+        action={testAction as any}
```

Comment:
cc @juliusmarminge types have changed so i just did `as any` 🙀 

maybe we fix this in a follow-up, wdyt?

Reply:
i've never used this `useAction` hook so I don't know what it's referring to